README.rst

```diff
@@ -123,6 +123,11 @@ How to Install:
             type="text/javascript"
             src="{% static "django_tables2_column_shifter/js/django_tables2_column_shifter.min.js" %}">
         </script>
+        <link
```

Comment:
I would rather to not add new static file to *column-shifter*. 
From my point of view, will be better if we add styles for not-hidable column as  a inline tag attribute like `style="...."` 

I propose: 

1. Extend table class with attribute `not_hidable_css`

```
class ColumnShiftTable(tables.Table):
    ....
    # Inline css style for not-hidable column (for button)
    not_hidable_column_css = "pointer-events:none; opacity:.6"
```

2. In template
```
    <li class="btn-shift-column"
         style="{% if not column in table.hideable_columns %}{{ table.not_hidable_column_css }}{% endif %}"
         data-td-class="{{ column.attrs.td.class }}"
         data-state="on"
         .......
    </li>
```

In this way, new static file won't be necessary and we keep reverse compatibility 
Of course, info about it should be added to readme.